src/components/ModalDrawer.js

```diff
@@ -0,0 +1,131 @@
+import _ from 'lodash/fp'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import RModal from 'react-modal'
+import { Transition } from 'react-transition-group'
+import { buttonPrimary, buttonSecondary, Clickable } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+RModal.defaultStyles = { overlay: {}, content: {} }
```

Comment:
This mutates the module, and we already do this in `Modal.js`, so we shouldn't need to do it again here. Actually, we should probably move this to `appLoader.js` along with the other global init stuff.